packages/fxa-auth-server/lib/server.js

```diff
@@ -294,6 +344,11 @@ async function create(log, error, config, routes, db, translator, statsd) {
     'setMetricsFlowCompleteSignal',
     metricsContext.setFlowCompleteSignal
   );
+  server.decorate(
+    'request',
+    'getAndSetMetricsEnabled',
+    metricsContext.getAndSetMetricsEnabled
+  );
```

Comment:
where is this used?

Reply:
This was left over from a different approach I was thinkign about, will remove.